library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -0,0 +1,152 @@
+# encoding: utf-8
+
+# ***************************************************************************
+#
+# Copyright (c) 2015 SUSE LLC
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail,
+# you may find current contact information at www.novell.com
+#
+# ***************************************************************************
+# File: fs_snapshot.rb
+#
+# Authors:
+#	Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+require "yast"
+
+module Yast2
+  # Represents the fact that Snapper is not configured for "/" (root).
+  class SnapperNotConfigured < StandardError
+    def initialize
+      super "Snapper is not configured."
```

Comment:
I expect the user of `FsSnapshot` to be a programmer. Also these messages are not translated. But you're right: I could make the programmer live easier if I point him in the right direction.